Packs/WatchguardFirebox/ModelingRules/WatchguardFirebox_1_3/WatchguardFirebox_1_3.xif

```diff
@@ -0,0 +1,23 @@
+[MODEL: dataset="watchguard_xtm_raw"]
+alter proto = uppercase(proto)
+| alter xdm.event.id = leefEventID,
+    xdm.observer.version = leefProductVersion,
+    xdm.source.port = to_integer(srcPort),
```

Comment:
@guytamir10 Isn't this field INT already?

Reply:
was fixed